classes/game.rb

```diff
@@ -0,0 +1,92 @@
+class Game
+  attr_reader :game_name, :player1, :player2, :players, :turn_of, :table
+
+  def initialize (game_name, player1, player2)
+    @game_name = game_name
+    @player1 = player1
+    @player2 = player2
+    @players = [player1, player2]
+
+    @game_set = ['_', '_', '_', '_', '_', '_', '_', '_', '_']
+    @victory_combinations = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+    ]
+
+    @turn_of = random_initial_turn_of
+    @table = set_table
+  end
+
+  def start_game
+    i=0
+
+    while i<9 do
+      i+=1
+      choice_of_player
+    end
+  end
```

Comment:
This is great. However, in languages like ruby and python (and even js) it's very common to see `while` or `for`loops; instead, [enumerables](https://ruby-doc.org/core-3.1.1/Enumerable.html) are used more often. This can change to something like
```
(0...9).each do |i|
  coice_of_player
end
```